recipes/scijava-jupyter-kernel/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -euo
+
+# Install Beakerx...
+# ...waiting for a Maven artifact.
+cd $RECIPE_DIR/
+
+git clone https://github.com/twosigma/beakerx.git
+cd beakerx/
+git checkout 585f07c5dfe7f9f0f97053d90ffc5a696d972382
```

Comment:
There are multiple sources in both `bld.bat` and `build.sh` I have confess that I am confused. Can you clarify what is being install and why? Can we factor out any of those as another package?